test/specs/footer/verifyAllElementsOnTheFooterDisplayed.e2e.ts

```diff
@@ -0,0 +1,122 @@
+import { expect } from '@wdio/globals'
+import homePage from '../../pageobjects/home.page';
+import privacyPolicyPage from '../../pageobjects/privacyPolicy.page';
+import cookiePolicyPage from '../../pageobjects/cookiePolicy.page';
+import termsConditionsPage from '../../pageobjects/termsConditions.page';
+import advertsPage from '../../pageobjects/adverts.page';
+import tendersPage from '../../pageobjects/tenders.page';
+
+describe('id:C214 - Verify that all elements on the footer are displayed and all links are clickable', () => {
+    it('1. Scroll down to the footer', async () => {
```

Comment:
Use one it-block for the entire test. Do not split it with it blocks

Reply:
done